copier/tools.py

```diff
@@ -103,15 +103,29 @@ def to_nice_yaml(data: Any, **kwargs) -> str:
     return result or ""
 
 
+def get_env(
```

Comment:
I guess this makes it more explicit. "Env" can mean many things.

```suggestion
def get_jinja_env(
```